templates/prescription/general_edit.html

```diff
@@ -107,7 +107,7 @@
     {if $DRUG_ARRAY_VALUES}
     &nbsp; &nbsp; &nbsp; &nbsp;
         {if $prescription->get_refills() >= $prescription->get_dispensation_count()}
-            <input type="submit" name="disp_button"class='btn btn-primary btn-sm'  style="margin:0 5px 0 2px;" value="{xla t='Save and Dispense'}" />
+            <input type="submit" name="disp_button"class='btn btn-primary btn-sm my-0 mr-5 ml-2' value="{xla t='Save and Dispense'}" />
```

Comment:
The padding/margin in BS4 isn't measured in pixels. I suggest you look at documentation again as there is too much spacing here.